site/gatsby-site/src/templates/cite.js

```diff
@@ -183,68 +225,64 @@ export default class IncidentCite extends Component {
       return null;
     }
     const {
-      allMdx,
-      allMongodbAiidprodIncidents: {
-        group
-      },
-      site: {
-        siteMetadata: { docsLocation, title },
-      },
+      // allMdx,
+      allMongodbAiidprodIncidents: { group },
     } = data;
 
-    const gitHub = require('../components/images/github.svg');
+    // const navItems = allMdx.edges
```

Comment:
What is going on here?

Reply:
This var has never been used, I kept it for now to have your confirmation on removal.

Reply:
Ok, if it was never read go ahead and remove it. Is this the case in some of the other linted files?

Reply:
Yes

Reply:
Removed.